build/vsts-ci.yml

```diff
@@ -107,6 +107,10 @@ phases:
       artifactName: PackageAssets
       artifactType: container
 
+  # Terminate all dotnet build processes.
```

Comment:
>processes [](start = 31, length = 9)

I think you also need to disable that two LightGBM tests on x86 otherwise you wouldn't pass build.

Reply:
Ok thank you will do that!